### PR TITLE
Merchant Can Fulfill Orders US49-51 SM & ZH

### PR DESCRIPTION
**Pull request by steve-meyers**
- Add sell_stock Item model method
- Add functionality to fulfill item_orders
- Remove Cancel Order from Merchant Order Show Page
- [x] All tests passing?

### CODE REVIEW

File: spec/features/merchant_dashboard/orders_show_spec.rb [hunk 1]
```diff
@@ -65,4 +65,192 @@
...
+      visit "/merchant/orders/#{order1.id}"
+      
+      within("#item-#{pull_toy.id}") do
+        expect(page).to_not have_button "Fulfill"
+        expect(page).to have_content("You do not have enough stock to fulfill this order!")
+      end
+    end
+    
+  end
 end
```
Comment (elguapogordo) [inline spec/features/merchant_dashboard/orders_show_spec.rb:256]: that is a girthy test!

**Review by elguapogordo — APPROVED** [review]

File: app/controllers/merchant/item_orders_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Merchant::ItemOrdersController < ApplicationController
+  def update
+    item_order = ItemOrder.find(params[:id])
+    fulfill(item_order)
+    redirect_to "/merchant/orders/#{item_order.order_id}"
+  end
+  
+  private
+  
+  def fulfill(item_order)
```
Comment (jrsewell400) [inline app/controllers/merchant/item_orders_controller.rb:10]: Nice job keeping this to 5 lines!

File: spec/features/merchant_dashboard/orders_show_spec.rb (hunk 1 above)
Comment (jrsewell400) [reply spec/features/merchant_dashboard/orders_show_spec.rb:256]: yeah, damn!

**Review by jrsewell400 — APPROVED** [review]
All looks good to me!

**Pull request merged**